app/views/books/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= @book.title %>
+<h2>highest rating: <%= @book.highest_rating %></h2>
+highest rating Review: <%= Review.find_by(rating: @book.highest_rating).body %><br>
+highest rating User: <%= User.find_by(id: (Review.find_by(rating: @book.highest_rating).user_id)).name %>
```

Comment:
Whenever you find yourself starting to call straight to a class method in your view, take that as a red flag. This functionality should be built out on the model.

Reply:
Rather than going from Review on the class level, I'd want to build out something on an instance of Book. Something like:
```
def highest_rated
   reviews.order(:rating).last 
end
```
This would return me a review instance for the highest rated review for a specific book.